ibm/service/vpc/resource_ibm_is_backup_policy_plan.go

```diff
@@ -0,0 +1,532 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func ResourceIBMIsBackupPolicyPlan() *schema.Resource {
+	return &schema.Resource{
```

Comment:
Can you test the scenario of creating a single backup policy and multiple plan (use count = 2) at same time (just to see if mutex is required or not)

Reply:
Have tested 3,4 and 5(for validation purposes) plans for a policy multiple  using terraform configuration. None of the resource got stuck due to race condition.